docs/protocol/extension_generate.md

```diff
@@ -87,10 +87,12 @@ return an error.
 
     $generate_request =
     {
+      "id" : $string, #optional
```

Comment:
Please update the copyright at the top of the file from this:
```
# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
to this:
```
# Copyright 2023-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Reply:
Updated